src/types/type/app.type.ts

```diff
@@ -0,0 +1,35 @@
+export type AFTER_CAPTURE = 'SPECIES_INFO' | 'POINT_REGISTER' | 'POLYGON_REGISTER' | 'SAMPLE_TREE' | 'EDIT_INTERVENTION' | 'EDIT_SAMPLE_TREE' | 'PLOT_IMAGE' | 'REMEASUREMENT_IMAGE'
+
+export type INTERVENTION_TYPE = 'single-tree-registration' | 'multi-tree-registration' | 'removal-invasive-species' | 'fire-suppression' | 'fire-patrol' | 'fencing' | 'marking-regenerant' | 'liberating-regenerant' | 'grass-suppression' | 'firebreaks' | 'assisting-seed-rain' | 'soil-improvement' | 'stop-tree-harvesting' | 'direct-seeding' | 'enrichement-planting' | 'other-intervention' | 'maintenance'
```

Comment:
Is it possible to fix some wrong spellings like `enrichment-planting` or is this already part of the backend database?